Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -95,31 +99,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 
 extension AppDelegate: UNUserNotificationCenterDelegate {
-    // Delegate called when user responds to a notification. Set delegate in
-    // application:didFinishLaunchingWithOptions: method.
+    // Function called when a push notification is clicked or swiped away.
     func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        let handled = MessagingPush.shared.userNotificationCenter(
-            center,
-            didReceive: response,
-            withCompletionHandler: completionHandler
+        // Track a Customer.io event for testing purposes to more easily track when this function is called.
+        CustomerIO.shared.track(
+            name: "push clicked",
+            data: ["push": response.notification.request.content.userInfo]
         )
 
-        // If the Customer.io SDK does not handle the push, it's up to you to handle it and call the
-        // completion handler. If the SDK did handle it, it called the completion handler for you.
-        if !handled {
-            completionHandler()
-        }
+        completionHandler()
```

Comment:
I am not 100% sure for this particular method. For other methods in the `UserNotification` framework, if you do not call the completionhandler, you will have consequences such as a long delay for push notifications to be displayed on the device, for example. 

Because this method's job is only to handle when a push getting clicked, I am not sure if there are any consequences? At least, the iPhone user may not notice any weird behavior like they would if a push had a long delay to be displayed.